Scripts/script-ExportToCSV.yml

```diff
@@ -0,0 +1,74 @@
+commonfields:
+  id: da9e2f98-05c4-408b-84d1-21960f6dbad9
+  version: -1
+name: ExportToCSV
+script: |-
+  function convertToCSV(objArray) {
+      var array = typeof objArray !== 'object' ? JSON.parse(objArray) : objArray;
+      var str = '';
+      var line = '';
+      var headersInOrder = [];
+
+      // export the headers of csv
+      if (array.length > 0) {
+          line = '';
+          for (var property in array[0]) {
+              if (array[0].hasOwnProperty(property)) {
+                  if (line !== '') {
+                      line += ',';
+                  }
+                  line += property;
+                  headersInOrder.push(property)
+              }
+          }
+          str += line + '\r\n';
+      }
+
+      for (var i = 0; i < array.length; i++) {
+          line = '';
+          for (var headerIdx = 0; headerIdx < headersInOrder.length; headerIdx++) {
+              header = headersInOrder[headerIdx];
+              if (line !== '') {
+                  line += ',';
+              }
+
+              line += array[i][header];
+          }
+
+          str += line + '\r\n';
+      }
+
+      return str;
+  }
+
+  if (!Array.isArray(args.csvArray)) {
+      return {
+          Type: entryTypes.error,
+          ContentFormat: formats.text,
+          Contents: 'The csvArray that was passed was not array!'
+      }
+  }
+
+  var csvString = convertToCSV(args.csvArray);
+  var createdFileID = saveFile(csvString);
+
+  return {
+      Type: 9,
+      FileID: createdFileID,
+      File: args.fileName,
+      Contents: args.fileName
```

Comment:
Because both return args.fileName. I put HumanReadable only when there is a difference between Contents and HumanReadable. Should we put always the HumanReadable?